portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.servlets;
+public class Comment {
+  public long id;
+  public String comment;
+  public long timestamp;
+
+  public Comment(long id, String comment, long timestamp){
+    this.id=id;
```

Comment:
There should always be 1 space before and after the `=`, so for example this should be `this.id = id;`

Reply:
Addressed